### PR TITLE
fix(curriculum): improve description of City Skyline step 51

### PR DESCRIPTION
**Pull request by WilliamSpanfelner**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-51
 
 # --description--
 
-Gradient transitions often gradually change from one color to another. You can make the change a solid line like this:
+Gradient transitions often gradually change from one color to another. You can make the gradual change become a solid line like this:
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md:10]:
```suggestion
Gradient transitions often gradually change from one color to another. You can make the transition with a hard stop like this:
```
Maybe something like this? _Hard transition_ or _hard stop_ seem to be the wording used to describe the result.

Comment (WilliamSpanfelner) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md:10]: That is good, but what do you think of this:
Gradient transitions often gradually change from one color to another. When a more abrupt change is required, the transition can be made with a "hard stop" like this:

**Review by WilliamSpanfelner — COMMENTED** [review]
Please review amended change.

**Review by WilliamSpanfelner — COMMENTED** [review]
Please check changes and commit

**Review by WilliamSpanfelner — COMMENTED** [review]
Cool.

**Review by huyenltnguyen — APPROVED** [review]

**Review by gikf — APPROVED** [review]

**Pull request merged**